cypher/cypher-docs/src/docs/dev/ql/administration/security/deny-create-syntax.asciidoc

```diff
@@ -0,0 +1,9 @@
+.Command syntax
+[source, cypher]
+-----
+DENY CREATE ON GRAPH[S] { * | graph-name[,...] }
+    [ ELEMENT[S] { * | label-or-rel-type-name[,...] }
```

Comment:
The formatting here is a little hard to read. We did it in a more open way in the fine-grained read examples.